### ISSUE TITLE
Target size minimum lacks guidance in the Understanding document on inline text

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3500]
During the formation of the SC, there was _a lot_ of discussion on inline text and how to formulate wording that guided teams to how to determine when a link needed to have sufficient vertical space.

There is little about that in the [understanding document](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html). All it says is
> Inline: The Success Criterion does not apply to inline targets in sentences, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, guaranteeing that undersized links in adjacent lines of text fulfill the spacing exception (their 24 CSS pixel diameter circle don't intersect any other links or their circles) would require a large line height which can be undesirable in many design contexts.

I think we need an illustration on how links are not sufficiently spaced in common text presentations, as well as examples of demarcating when a link's "size is otherwise constrained by the line-height of non-target text" and where a link needs to meet the spacing requirement.

**Comment by patrickhlauke** [comment]
I can probably try to tackle some visuals...

**Comment by mbgower** [comment]
Thanks, Patrick. Once you are have something ready to look at, change the WCAG 2.x project status to "Drafted" (just underneath the Labels section, and we will review.

**Comment by dan-tripp-siteimprove** [comment]
I'd like to give this a +1.  And ask: until the Understanding doc gets updated, is there any source of guidance on the inline exception?   I opened issue https://github.com/w3c/wcag/issues/3898 and then discovered a certain [google doc](https://docs.google.com/document/d/1BFNDFnbU9CizswP4oEICpFHeqt7qXg9kRt2QEDoUXSM/edit#) which seems to be part of the discussion during the formation of the SC that mbgower mentioned above.  But then I discovered some [apparent contradictions in that google doc](https://github.com/w3c/wcag/issues/3898#issuecomment-2307525874), so now once again I don't know whether those cases - which are common, in the wild - pass or fail.  If anyone has any links which could clear this up, please let me know.
